packages/shell/src/util/versionOverride.ts

```
@@ -0,0 +1,235 @@
+ * Overrides whose write to sessionStorage FAILED — an embedded frame with
+ * storage denied, or a full quota. They stand in for the storage row so THIS
+ * document still resolves the pick (a deep link would otherwise seed nothing
+ * and the reconciliation would find no override at all).
+ *
+ * The mirror only ever ADDS: an entry that survives in storage is what the
+ * next boot resolves, so a delete that failed to persist must keep reading as
+ * present. The failing-override drop path depends on that — it verifies the
+ * clear before reloading, and a mirror that masked the surviving storage row
+ * would turn a bounded reload into a loop.
+ */
+let memoryOverrides: Record<string, AppVersionOverride> = {};
```

ESLint

[error] 'memoryOverrides' is never reassigned. Use 'const' instead.

(prefer-const)